solutionbox/structured_data/mltoolbox/classification/dnn/_classification_dnn.py

```diff
@@ -16,6 +17,45 @@ def train(train_dataset,
           job_name=None, 
           cloud=None, 
           ):
+  """Blocking version of train_async. See documentation for train_async."""
+  job = train_async(
+      train_dataset=train_dataset,
+      eval_dataset=eval_dataset,
+      analysis_output_dir=analysis_output_dir,
```

Comment:
Can you call this just analysis_dir? The current name makes it sound like this is an output from train, rather than an input to train.